fdbclient/ServerKnobs.cpp

```diff
@@ -851,6 +854,7 @@ void ServerKnobs::initialize(Randomize randomize, ClientKnobs* clientKnobs, IsSi
 	init( PEER_DEGRADATION_CONNECTION_FAILURE_COUNT,               5 );
 	init( WORKER_HEALTH_REPORT_RECENT_DESTROYED_PEER,           true );
 	init( STORAGE_SERVER_REBOOT_ON_IO_TIMEOUT,                 false ); if ( randomize && BUGGIFY ) STORAGE_SERVER_REBOOT_ON_IO_TIMEOUT = true;
+	init( WORKER_START_STORAGE_DELAY,                            0.0 ); if ( randomize && BUGGIFY ) WORKER_START_STORAGE_DELAY = 1.0;
```

Comment:
maybe around 1ms? do you intend to `delay(0.0)`?

Reply:
We don't want any delay when WORKER_START_STORAGE_DELAY is 0. When this is in use, we need a much longer delay, e.g. 10min, to prevent OOM.